mineinabyss-features/src/main/kotlin/com/mineinabyss/invisibleItemFrames/InteractionListener.kt

```diff
@@ -0,0 +1,40 @@
+package com.mineinabyss.invisibleItemFrames
+
+import com.destroystokyo.paper.event.entity.ProjectileCollideEvent
+import com.mineinabyss.idofront.messaging.broadcast
+import net.bytebuddy.implementation.bytecode.Throw
+import org.bukkit.entity.Entity
+import org.bukkit.entity.Item
+import org.bukkit.entity.ItemFrame
+import org.bukkit.entity.ThrownPotion
+import org.bukkit.event.EventHandler
+import org.bukkit.event.Listener
+import org.bukkit.event.entity.ProjectileHitEvent
+import org.bukkit.potion.PotionEffect
+import org.bukkit.potion.PotionEffectType
+import org.jetbrains.exposed.sql.intLiteral
+import org.jetbrains.exposed.sql.intParam
+
+class InteractionListener : Listener {
+    @EventHandler
+    fun ProjectileHitEvent.onDouseItemFrame() {
+        val entity = entity as? ThrownPotion ?: return
+
+        if (hitEntity !is ItemFrame) return
+        if (
+                entity
+                    .potionMeta
+                    .basePotionData
+                    .type
+                    .effectType != PotionEffectType.INVISIBILITY
+        ) return
```

Comment:
might have been the automatic formatter but this looks kinda bad. reformat it to essentially be just a line.
better to have 1 long line than using 7

Reply:
[24b400c](https://github.com/MineInAbyss/MineInAbyss/pull/158/commits/24b400c16407c2dda19deffb4a19936e90c6aae4) fixes this